src/phoenix/server/api/types/Dataset.py

```diff
@@ -238,25 +239,55 @@ async def experiments(
         return connection_from_list(data=experiments, args=args)
 
     @strawberry.field
-    async def experiment_annotation_names(self, info: Info[Context, None]) -> List[str]:
+    async def experiment_annotation_summaries(
+        self, info: Info[Context, None]
+    ) -> List[ExperimentAnnotationSummary]:
         dataset_id = self.id_attr
-        async with info.context.db() as session:
-            annotation_names = (
-                await session.scalars(
-                    select(distinct(models.ExperimentAnnotation.name))
-                    .join(
-                        models.ExperimentRun,
-                        models.ExperimentAnnotation.experiment_run_id == models.ExperimentRun.id,
-                    )
-                    .join(
-                        models.Experiment,
-                        models.ExperimentRun.experiment_id == models.Experiment.id,
+        query = (
+            select(
+                models.ExperimentAnnotation.name,
+                func.min(models.ExperimentAnnotation.score),
+                func.max(models.ExperimentAnnotation.score),
+                func.avg(models.ExperimentAnnotation.score),
+                func.count(),
+                func.sum(
```

Comment:
`count(error)` will skip the nulls
```suggestion
                func.count(models.ExperimentAnnotation.error)
```

Reply:
addressed in 39a6454fd64e857da7c096359911fa068d1e05be